addons/survey/static/tests/tours/survey_tour_session_manage.js

```diff
@@ -175,197 +230,72 @@ registry.category("web_tour.tours").add('test_survey_session_manage_tour', {
     run: nextScreen
 }, {
     trigger: 'h1:contains("Regular Simple Choice")',
+    extra_trigger: '.o_survey_session_progress_small[style*="width: 100%"]',
```

Comment:
As this is fairly specific, maybe add a small code comment that is similar to what you have in the commit message?